handlers/create.go

```diff
@@ -79,23 +83,42 @@ func CreateMachine(event *events.Event, apiClient *client.RancherClient) error {
 	return publishReply(reply, apiClient)
 }
 
-func logProgress(resourceId string, readerStdout io.Reader, readerStderr io.Reader) {
+func logProgress(readerStdout io.Reader, readerStderr io.Reader, uuid string, event *events.Event, apiClient *client.RancherClient) {
 	// We will just log stdout first, then stderr, ignoring all errors.
 	scanner := bufio.NewScanner(readerStdout)
 	for scanner.Scan() {
+		msg := scanner.Text()
 		log.WithFields(log.Fields{
-			"resourceId: ": resourceId,
-		}).Infof("stdout: %s", scanner.Text())
+			"resourceId: ": event.ResourceId,
+		}).Infof("stdout: %s", msg)
+		msg = filterDockerMessage(msg, uuid)
+		if msg != "" {
+			publishTransitioningReply(msg, event, apiClient)
+		}
 	}
 
 	scanner = bufio.NewScanner(readerStderr)
 	for scanner.Scan() {
 		log.WithFields(log.Fields{
-			"resourceId": resourceId,
+			"resourceId": event.ResourceId,
 		}).Infof("stderr: %s", scanner.Text())
 	}
 }
 
+func filterDockerMessage(msg string, uuid string) string {
+	// Docker log messages come in the format: time=<t> level=<log-level> msg=<message>
+	// We just want to return <message> to cattle and only messages that do not contain the machine uuid
+	re := regexp.MustCompile("msg=.*")
+	msgSlice := re.FindStringSubmatch(msg)
+	msgSlice = strings.Split(msgSlice[0], "=")
+	if strings.Contains(msgSlice[1], uuid) == true {
+		return ""
+	} else {
+		return strings.Trim(strings.TrimSpace(msgSlice[1]), "\"")
```

Comment:
I don't think you need Trim + TrimSpace. Just `strings.Trim(msgSlice[1], "\" ")` should do it. Take a look at the documentation and example for trim:
http://golang.org/pkg/strings/#Trim